src/illumidesk/authenticators/utils.py

```diff
@@ -105,5 +105,5 @@ def convert_request_to_dict(self, arguments: Dict[str, List[bytes]]) -> Dict[str
         """
         args = {}
         for k, values in arguments.items():
-            args[k] = values[0].decode() if len(values) == 1 else [v.decode() for v in values]
+            args[k] = values[0].decode() if len(values) >= 1 else [v.decode() for v in values]
```

Comment:
It seems like we're inly supporting a unique case: a list of key:value so why not change it to:
`args[k] = values[0].decode() ` ?

Reply:
Yeah, I thought of that too. I think it's a valid point since the LTI specification or other query params don't consider adding a list of items per argument (yet). I'll update and push again for approval.

Reply:
@netoisc I'm going to leave as-is. If we leave just `values[0].decode()` then we get out of range exceptions since in some cases the arguments can have empty/null values so an index list isn't established.